src/management/api/portal/general/apiPortal.controller.ts

```diff
@@ -158,6 +162,10 @@ class ApiPortalController {
     }
   }
 
+  initializeApiLabelsDictionary() {
+    this.PortalConfigService.get().then(response => this.apiLabelsDictionary = response.data.api.labelsDictionary);
```

Comment:
Since the portal configuration is loaded once in the `Constants` variable at startup, you just need to write something like that:
`this.apiLabelsDictionary = this.Constants.api.labelsDictionary`

Reply:
resolved